etp-backend/src/main/clj/solita/etp/service/laskutus.clj

```diff
@@ -293,6 +294,19 @@
                                :xlsx (.getName xlsx-file)
                                :pdf-result? pdf-exists?)))))))
 
+(defn send-tasmaytysraportti-email! [tasmaytysraportti-file]
+  (smtp/send-email! config/smtp-host
```

Comment:
packing these SMTP parameters to a map or a record might help a bit with readability.

Reply:
Thought about that too. But then realized that there are probably use cases where someone wants to another function (with partial) where host, port, etc are already bound. That's not possible (with partial at least) if arguments are inside a map.